stdlib/multiprocessing/popen_spawn_win32.pyi

```diff
@@ -0,0 +1,30 @@
+import sys
+from multiprocessing.process import BaseProcess
+from typing import ClassVar
+
+from .util import Finalize
+
+if sys.platform == "win32":
+    __all__ = ["Popen"]
+
+    TERMINATE: int
+    WINEXE: bool
+    WINSERVICE: bool
+    WINENV: bool
+
+    class Popen:
+        finalizer: Finalize
+        method: ClassVar[str]
+        pid: int
+        returncode: int | None
+        sentinel: int
+
+        def __init__(self, process_obj: BaseProcess) -> None: ...
+        def duplicate_for_child(self, handle: int) -> int: ...
+        def wait(self, timeout: float | None = ...) -> int | None: ...
+        def poll(self) -> int | None: ...
```

Comment:
I don't think `wait` (and, by extension, `poll`, which returns whatever `wait` returns) can return `None`: if `self.returncode` is `None` when `wait` is called, there's this whole branch of logic which looks like it's dedicated to guaranteeing `self.returncode` is set to an `int` before `self.returncode` is returned: https://github.com/python/cpython/blob/2fbee85931296bbeddae6358583e400ce5321f89/Lib/multiprocessing/popen_spawn_win32.py#L102-L113

```suggestion
        def wait(self, timeout: float | None = ...) -> int: ...
        def poll(self) -> int: ...
```

Reply:
Oh but on the other hand there's this `if self.wati(timeout=1.0) is None` check here: https://github.com/python/cpython/blob/2fbee85931296bbeddae6358583e400ce5321f89/Lib/multiprocessing/popen_spawn_win32.py#L125

Reply:
Yeah ignore me, this suggestion is incorrect.

Reply:
All good :) I had the same line of thought when I was putting it together

Reply:
Glad we could go on this journey together 😄